COMMUNITY_MEMBERSHIP.md

```diff
@@ -81,7 +91,7 @@ Members are continuously active contributors in the community. They can have iss
 
 Code approvers are able to both review and approve code contributions, as well as help maintainers triage issues and do project management. While code review is focused on code quality and correctness, approval is focused on holistic acceptance of a contribution including: backwards/forwards compatibility, adhering to API and flag conventions, subtle performance and correctness issues, interactions with other parts of the system, etc.
 
-*Defined by:*  [Approvers of the Keptn GitHub organization](https://github.com/orgs/keptn/teams/approvers), and [CODEOWNERS](https://help.github.com/en/articles/about-code-owners) in [Keptn](https://github.com/keptn/keptn)
+*Defined by:* Member of the subproject approvers team (https://github.com/orgs/keptn/teams/<subproject>-approvers) and listed in the subprojects `teams.yaml` within [Peribolos Configuration Directory][Peribolos Configuration Directory]
```

Comment:
same thing here with the half-rendered link

Reply:
thank you changed it to code